example/README.md

```diff
@@ -10,20 +10,17 @@
 ### Install
 1. Clone the `stripe-android` repository.
 2. Open the project in Android Studio.
-3. After [deploying the example backend to Heroku](#deploy-the-example-backend-to-heroku) and [configuring the app](#configure-the-app), build and run the project.
+3. After [remixing the Glitch project](#remix-the-example-project-on-glitch) and [configuring the app](#configure-the-app), build and run the project.
```

Comment:
This is now an optional step, as people can just use our example Glitch, right?

Reply:
No, they don't have our publishable key. In the future we want to return it in the backend call, like we do in the paymentsheet-example app. For now this is just migrating off Heroku.